Resources/AzureDevOps/azure-pipelines.yml

```diff
@@ -10,29 +10,33 @@ resources:
   - repository: dfc-devops
     type: github
     name: SkillsFundingAgency/dfc-devops
-    ref: refs/tags/v1.2.2
+    ref: refs/tags/v1.6.12
     endpoint: 'GitHub (ESFA)'
 
 jobs:
 
-# Build the app with the dfc-dotnetcore-build template.  Runs if the output of TestBranchName is Version2+
+- template: AzureDevOpsTemplates/Build/dfc-arm-build.yml@dfc-devops
+  parameters:
+    ArmTemplateRoot: $(ArmTemplateRoot)
+
+# Build the app with the dfc-dotnetcore-build template
 - job: BuildDotNetStandard
   displayName: Build-DotNetStandard
   pool:
-    name: Continuous Integration 02 - SSD - 160ACU
+    name: NCS - CI and CD
     demands:
     - msbuild
     - visualstudio
 
   steps:
-  - template: AzureDevOpsTemplates/Build/dfc-arm-build.yml@dfc-devops
```

Comment:
we'll still need the ARM build step,  which needs to come before dfc-dotnetcore-build-sonar step.